aio/src/app/custom-elements/events/events.component.spec.ts

```diff
@@ -31,162 +29,267 @@ describe('EventsComponent', () => {
 
   describe('ngOnInit', () => {
     it('should have 2 upcoming events', () => {
```

Comment:
This description does not help understand what is being tested (why 2). Please change it to something like:

```suggestion
    it('should have recognize upcoming events', () => {
```

Same for other tests below.